templates/rust/Cargo.ejs.toml

```diff
@@ -0,0 +1,43 @@
+[package]
+name = "<%= name %>"
+version = "0.1.0"
+authors = ["<%= orgname %>"]
```

Comment:
Oh not only ejs files can be used with markdowns but toml files as well. This is a very good example of the toml file with ejs.

Reply:
Any file of any type can be used as EJS template. So long as it's text.

Not just that amigo. Shoveler for now only use basic EJS capabilities. The engine is a powerful one, allowing Javascript logic to be placed and executed within any raw text files. To output the result of the evaluation in place of the placeholder.

So, EJS allows us to make a readme that dynamically generates commands custom to say Nodejs, rust, or Python. All that out of a single Readme template rather having to place a readme in each folder that really would be 90% duplication.

I suggest you pick up a feature to make that work to grasp the power of EJS and make shoveler shoveling better. It's easy.

Maybe come up with some cool thing it could do. The doc is there:
https://ejs.co/